npm/react/cypress/component/basic/rerender/effects.cy.jsx

```diff
@@ -60,8 +59,8 @@ it('should run unmount effect cleanup when unmounting', () => {
       expect(effectCleanup).to.have.been.callCount(0)
     })
 
-    cy
-    .then(() => ReactDom.unmountComponentAtNode(getContainerEl()))
+    // mount something else to trigger an unmount event
+    cy.mount(<div>Hello </div>)
```

Comment:
Are we able to use [root.unmount](https://react.dev/reference/react-dom/client/createRoot#root-unmount) instead?

Reply:
Hmm, since `unmount` has been removed, should we still be exporting [getContainerEl](https://github.com/cypress-io/cypress/blob/develop/npm/react18/src/index.ts#L83) here then?

Reply:
We should probably just remove the `unmount` function entirely. I do wonder though if we should just expose `root`.

Reply:
> Hmm, since `unmount` has been removed, should we still be exporting [getContainerEl](https://github.com/cypress-io/cypress/blob/develop/npm/react18/src/index.ts#L83) here then?

it only makes sense if you are removing the container element from the dom, but that isn't exactly the same thing as `unmount` .I don't see any harm in exposing the root, but I am not sure what the initial design decision was to not expose it in the first place 🤷🏻

Reply:
Here are our migration docs for unmount: https://docs.cypress.io/app/references/migration-guide#React---unmount-Removed

Reply:
Since `ReactDOM.unmountComponentAtNode` is deprecated in 18 and removed in 19, I think we change this `getContainerEl` to `getReactRoot` or something similar. That seems appropriate at least to me. However I don't want to do that in this PR since its out of scope from the cutover, but I can create a follow up issue that I can PR into Cypress 14 that does this

Reply:
https://github.com/cypress-io/cypress/issues/30622